tests/time_tests/scripts/run_timetest.py

```diff
@@ -49,11 +49,17 @@ def parse_stats(stats: list, res: dict):
                     parse_stats(v, res)
 
 
-def aggregate_stats(stats: dict):
+def aggregate_stats(stats: dict, args:dict):
     """Aggregate provided statistics"""
-    return {step_name: {"avg": statistics.mean(duration_list),
+    agg_stats = {step_name: {"avg": statistics.mean(duration_list),
                         "stdev": statistics.stdev(duration_list) if len(duration_list) > 1 else 0}
             for step_name, duration_list in stats.items()}
+    if args["model_cache"]: # FIL-First Inference Latency
```

Comment:
Do not need this as FIL is prepared in .cpp file below

Reply:
agree, removed